### PR TITLE
feat: adding save draft functionality

### PR DESCRIPTION
**Pull request by kiram15**
# For all changes

- [x] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

**Review by johnnagro — APPROVED** [review]

File: src/components/settings/SettingsLMSTab/ExistingLMSCardDeck.jsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const ExistingLMSCardDeck = ({
 
   // Map the existing config data to individual cards
   const listItems = configData.map((config) => (
-    <Card isClickable tabIndex="0" className="p-2.5 existing-lms-card-width" key={config.channelCode + config.id}>
```
Comment (alex-sheehan-edx) [inline src/components/settings/SettingsLMSTab/ExistingLMSCardDeck.jsx:71]: thx for the fix 👍

File: src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx [hunk 2]
```diff
@@ -97,34 +101,21 @@ describe('<BlackboardConfig />', () => {
     };
     expect(LmsApiService.updateBlackboardConfig).toHaveBeenCalledWith(expectedConfig, 1);
   });
-  test('it creates new configs on submit', () => {
```
Comment (alex-sheehan-edx) [inline src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx:100]: I know they're accomplishing kinda the same thing, but we should probably keep the test of the submitting a new config as well as have this new test, or am I crazy?

**Review by alex-sheehan-edx — APPROVED** [review]
One testing q but otherwise looks good!

**Comment by alex-sheehan-edx** [comment]
just confirming, we'd want to ship the backend change before this goes out yes?

**Pull request merged**